custom/icds_reports/static/js/directives/adolescent-girls/adolescent-girls.directive.js

```diff
@@ -3,10 +3,10 @@ var url = hqImport('hqwebapp/js/initial_page_data').reverse;
 
 function AdolescentWomenController($scope, $routeParams, $location, $filter, demographicsService, locationsService,
     dateHelperService, navigationService, userLocationId, storageService, haveAccessToAllLocations,
-    baseControllersService, isAlertActive, isMobile) {
+    baseControllersService, isAlertActive, isMobile, haveAccessToFeatures) {
     baseControllersService.BaseController.call(this, $scope, $routeParams, $location, locationsService,
         dateHelperService, navigationService, userLocationId, storageService, haveAccessToAllLocations,
-        false, isMobile);
+        false, isMobile, haveAccessToFeatures);
```

Comment:
why is this name generic `Features`? I would think of this name to be something like or better than `haveAccessToBeta` or `showOutOfSchoolData`

Reply:
Hey yeah, this is because this is the general feature flag of the dashboard, we reuse this feature whenever we release any new feature

Reply:
a better name might be then `haveAccessToBetaFeatures`

Reply:
this name bugs me too. but not enough to change it in the 8 billion places we'd need to.